docs/data/material/guides/interoperability/interoperability.md

```diff
@@ -614,14 +614,16 @@ If you are used to Tailwind CSS and want to use it together with the MUI compone
 If you use a different framework, or already have set up your project, follow these steps:
 
 1. Add Tailwind CSS to your project, following the instructions in https://tailwindcss.com/docs/installation.
-2. Remove Tailwind's `base` directive in favor of the `CssBaseline` component provided by `@mui/material`, as it plays nicer with the MUI components.
+2. Remove Tailwind CSS's preflight style as it applies global changes that break MUI components. If you want to retrain the notion of preflight, you can use the [`CssBaseline` component](/material-ui/react-css-baseline/) that better isolates global resets. More details on [Tailwind CSS preflight styles](https://tailwindcss.com/docs/preflight).
```

Comment:
**Off topic**: we could look into why it's problematic. AFAIK, with Bootstrap, we apply CSS to reset the resets of Bootstrap, we could do the same with Tailwind CSS, allowing us to use all the MUI components in an existing codebase.